pkg/controllers/awsloadbalancercontroller/deployment.go

```diff
@@ -147,6 +164,21 @@ func desiredDeployment(name, namespace, image, vpcID, clusterName, awsRegion, cr
 								},
 							},
 						},
+						{
+							Name: boundSATokenVolumeName,
+							VolumeSource: corev1.VolumeSource{
+								Projected: &corev1.ProjectedVolumeSource{
+									DefaultMode: aws.Int32(420),
```

Comment:
Make this a constant. Also you shouldn't be using the aws pointer helper function. Kubernetes has it's own pointer helper called `pointer.Int32()`.

Reply:
Fixed.